src/commands/generate.ts

```diff
@@ -29,6 +29,8 @@ export class GenerateCommand extends BaseCommand {
             'Source fhir packages (example: hl7.fhir.r4.core@4.0.1)',
         ).conflicts('files');
 
+        const defaultPathToSchema = './tmp/type-schema.ndjson';
```

Comment:
Better, if we build cache name from the user input. E.g. md5 from package-names and fhir-schema names with some prefix. In this case it will use not only "last" execution, but all history runs.